config/detekt/detekt.yml

```diff
@@ -382,6 +404,10 @@ formatting:
 
 naming:
   active: true
+  BooleanPropertyNaming:
+    active: false
```

Comment:
active: false?

Reply:
Yes, we have properties violating this rule. For example `warningsAsErrors`